src/styles/product-card.scss

```
@@ -0,0 +1,109 @@
+$main-color: #060b35;
+$secondary-font-size: 10px;
+$secondary-color: #616070;
+$line-height-bigger: 18px;
+$line-height-smaller: 14px;
+$btn-color: #00acdc;
+
+.product-card {
+  box-sizing: border-box;
+
+  padding: 32px 16px 16px;
+  width: 200px;
+
+  border: 1px solid #f3f3f3;
+  font-family: Roboto, Arial, sans-serif;
+  font-weight: 400;
+  color: $main-color;
+  font-size: 12px;
+  &__image {
+    box-sizing: border-box;
+    display: block;
+
+    margin: 0 auto 40px;
+
+    height: 134px;
+
+
+    background-size: cover;
```

```suggestion
  font-size: 12px;
 
   &__image {
    box-sizing: border-box;
    display: block;
    margin: 0 auto 40px;
    height: 134px;
    background-size: cover;
```
Add a blank line between blocks. You can also add blank lines between some property blocks for example all properties related to position then blank line, then another block, but only one

Fix in all places